locksmith/__tests__/controllers/eventController.test.ts

```diff
@@ -108,6 +108,38 @@ describe('Event Controller', () => {
       })
     })
   })
+  describe('event update', () => {
+    describe('when unaccompanied by a valid signature', () => {
```

Comment:
How would you feel about adding a test case here for the scenario where the signature is valid but the person is attempting to update a lock that they may not own?

Reply:
I'd feel good about it! Test added.